atomkraft/utils/project.py

```diff
@@ -14,16 +17,34 @@ def __init__(self) -> None:
         super().__init__("Outside of Atomkraft project")
 
 
+def copy_file_from_template(file_path: Path, project_path: Path):
+    package_path = imp.find_module("atomkraft")[1]
+    file_path_abs = os.path.join(package_path, "templates/project", file_path)
+    dest_path = os.path.join(project_path, file_path)
+
+    os.makedirs(os.path.dirname(dest_path), exist_ok=True)
+    shutil.copyfile(file_path_abs, dest_path)
+
+
 def project_root() -> Path:
     cwd = Path(os.getcwd())
     while cwd != cwd.parent:
-        if (
-            (cwd / "pyproject.toml").exists()
-            and (cwd / "atomkraft.toml").exists()
-            and (cwd / ATOMKRAFT_INTERNAL_DIR / "config.toml").exists()
-        ):
+        if (cwd / "atomkraft.toml").exists():
+            if not (cwd / "pyproject.toml").exists():
+                logging.warn(
+                    f"Configuration file pyproject.toml not found, populating in root {cwd}"
+                )
+                # copy pyproject.toml file from template
+                copy_file_from_template("pyproject.toml", cwd)
+            if not (cwd / ATOMKRAFT_INTERNAL_DIR / "config.toml").exists():
+                logging.warn(
+                    f"Configuration file config.toml not found, populating in {cwd/ATOMKRAFT_INTERNAL_DIR}"
+                )
+                # copy config.toml file from template
+                copy_file_from_template(os.path.join(ATOMKRAFT_INTERNAL_DIR, "config.toml"), cwd)
```

Comment:
I would prefer a situation where these files not generated.

If a user just have an `atomkraft.toml` with their chain configuration and just wants to spawn a local testnet, they don't really need `.pyproject.toml` and `config.toml`